test/server/lib/DocApi.ts

```diff
@@ -375,6 +375,54 @@ function testDocApi() {
         }
       });
     }
+
+    it(`POST /api/docs ${content} can create saved docs in workspaces`, async function () {
+      // Make a workspace.
+      const chimpyWs = await userApi.newWorkspace({name: "Chimpy's Workspace"}, ORG_NAME);
+
+      // Create a document in the new workspace.
+      const user = chimpy;
+      const body = {
+        documentName: "Chimpy's Document",
+        workspaceId: chimpyWs,
```

Comment:
Can you add another test with a workspaceId that the user is not an editor/owner of, and check that document creation is denied?

Reply:
(for both the case of a fresh new document, and an uploaded document)